keepassxc-browser/options/options.js

```diff
@@ -118,6 +118,18 @@ options.initGeneralSettings = function() {
     $('#blinkMinTimeout').val(options.settings['blinkMinTimeout']);
     $('#allowedRedirect').val(options.settings['allowedRedirect']);
 
+    chrome.commands.getAll(function(commands) {
```

Comment:
Please do not use `chrome` namespace. Browser-polyfill ensures you can use `browser` everywhere.